plugins/woocommerce/includes/admin/settings/class-wc-settings-payment-gateways-react.php

```diff
@@ -79,6 +132,40 @@ protected function run_gateway_admin_options( $gateway ) {
 	public function get_sections() {
 		return array();
 	}
+
+	/**
+	 * Save settings.
+	 */
+	public function save() {
```

Comment:
Copied back from the original payments class, saving a section wasn't possible prior